README.md

```diff
@@ -7,16 +7,15 @@
 
 ## Installation
 
-The Nexmo CLI requires NodeJS 4 or above. If you don't have Node installed on your system, goto (<https://nodejs.org/en/download/>) and download the appropriate installer for your system.
+The Nexmo CLI requires Node.js 4 or above. If you don't have Node.js installed on your system, go to https://nodejs.org/en/download/ and download the appropriate installer for your system.
 
-Install the `nexmo-cli` from NPM.
+With Node.js installed, you can then install the `nexmo-cli` package from the npm registry.
 
 ```bash
-npm install nexmo-cli -g
+$ npm install nexmo-cli -g   # Using npm
```

Comment:
Can we add a link to https://docs.npmjs.com/getting-started/fixing-npm-permissions instead?

Reply:
Added!